src/main/resources/templates/db_scripts/schema.json

```diff
@@ -43,6 +43,19 @@
       "withMetadata": true,
       "populateJsonWithId": true
     },
+    {
+      "tableName": "interface_credentials",
+      "generateId": true,
+      "fromModuleVersion": 1.0,
+      "withMetadata": true,
+      "populateJsonWithId": true,
```

Comment:
As with `generateId`, `populateJsonWithIdis` is redundant, as it creates a trigger in the database that tries to populate in the id every time a record is created/updated after PgUtils has already done so and from the next version of RMB it will not be supported as well.